app/controllers/users_controller.rb

```diff
@@ -18,8 +32,40 @@ def create
     end
   end
 
+  def update
+    @user = User.find_by id: params[:id]
+    if @user.update_attribute user_params
+      flash[:success] = t "profileupdated"
+      redirect_to @user
+    else
+      render "edit"
```

Comment:
```suggestion
      render :edit
```
Thêm flash edit thất bại